src/views/List.jsx

```diff
@@ -1,22 +1,49 @@
+import { useState } from 'react';
 import { ListItem } from '../components';
 
 export function List({ data }) {
+	const [filter, setFilter] = useState('');
```

Comment:
Amazing work! Everything works well despite my best efforts to break it.
My only suggestion for improved readability would be to name the state variable something other than `filter`, since the `filter` array method is being used lower down.
Nice job!

Reply:
Ah good point @mentalcaries ! I've now changed it from filter to listFilter.